### DISCUSSION TITLE
Can finetune whisper-large-v2 with data augmentation SpecAugment, Stochastic Depth, and BPE Dropout?

### DISCUSSION THREAD

**Discussion by maowandong** [openai/whisper#1547] [Q&A]
I'm fine-tuning Whisper and noticed that Whisper-large-v2, compared to other versions of Whisper, has been trained for 2.5X longer. It also includes data augmentation techniques such as SpecAugment, Stochastic Depth, and BPE Dropout. I wonder, doesn't this extended training time of 2.5X lead to overfitting? Also, are these data augmentation techniques suitable for use during fine-tuning? If they can be used, are there any recommended open-source implementations that I can refer to?

**Comment by sky-lc** [comment]
Continuous attention

**Comment by mizoru** [comment]
Yes, too bad the implementations have not been shared
